### DISCUSSION TITLE
github pages

### DISCUSSION THREAD

**Discussion by ADITor-99** [community/community#23274] [Other Feature Feedback, Questions, & Ideas]
<p>I uploaded the whole files after creating it on my desktop to the repository and when I view it on my desktop the css is linked properly but after deploying it to github pages the css links are not working please help anyone . Here is my repository link</p>
<p><a href="https://github.com/ADITor-99/Project0.git" rel="nofollow noopener">https://github.com/ADITor-99/Project0.git</a></p>

**Comment by thatjar** [comment, marked as answer]
<p>I think you haven’t linked th files properly. Here’s how you’re supposed to link them:</p>
<pre><code>&lt;link href="css.css" rel="stylesheet" type="text/css"&gt;
</code></pre>
<p>This line should be inside the &lt;head&gt; tag and below the &lt;title&gt; tag.</p>
<p>Make sure you replace css.css with the file name.</p>
